core/modules/taxonomy/taxonomy.api.php

```diff
@@ -272,6 +272,24 @@ function hook_taxonomy_term_view_alter(&$build) {
   $build['#post_render'][] = 'my_module_taxonomy_term_post_render';
 }
 
+/**
+ * Alter the results of taxonomy_term_page()
+ *
+ * This hook is called after the node listing for a taxonomy term page has
+ * been assembled in a structured array and may be used for altering that
+ * listing.
+ *
+ * @param $build
+ *   A renderable array representing the taxonomy term content.
+ *
+ * @see taxonomy_term_page()
+ */
+function hook_taxonomy_term_page_alter(&$build) {
+  // Remove the list of nodes with this term (build).
+  unset($build['nodes']);
+  // Remove the pager for the list of nodes.
+  unset($build['pager']);
+}
```

Comment:
Need blank line after this:
```suggestion
}

```